ts/Series/AreaRange/AreaRangeSeries.ts

```diff
@@ -267,23 +267,28 @@ class AreaRangeSeries extends AreaSeries {
      * @private
      */
     public translate(): void {
-        const series = this,
-            yAxis = series.yAxis;
+        const series = this;
 
         areaProto.translate.apply(series);
 
         // Set plotLow and plotHigh
         series.points.forEach(function (
-            point: AreaRangePoint
+            point: AreaRangePoint,
+            i: number
         ): void {
-
             const high = point.high,
                 plotY = point.plotY;
 
             if (point.isNull) {
                 point.plotY = null as any;
             } else {
+                const yAxis = series.chart.hasParallelCoordinates ?
+                    series.chart.yAxis[i] :
```

Comment:
Just a note: it's not a mistake and you can leave it as it is, but you could get rid of `i` parameter and use `point.index` (or something similar) instead ;)

Reply:
Nice one, thanks! 👍